core/prop/prop.go

```diff
@@ -0,0 +1,18 @@
+package prop
```

Comment:
File needs a package comment.

Reply:
Looks like this still needs to happen

Reply:
Comment is below deleted lines (new line 3)

Reply:
Ah, comment should be above the package:

```go
// Package prop does a thing
package prop
```

But at this point, it's optional, so easy to clean up later =).